resources/property.py

```diff
@@ -57,6 +56,27 @@ def post(self, id):
 
         return property.json(), 201
 
+class ArchiveProperties(Resource):
+    parser = reqparse.RequestParser()
+    parser.add_argument('ids', action='append')
+
+    @admin_required
+    def post(self):
+        data = ArchiveProperties.parser.parse_args()
+        propertyList = []
+
+        for id in data['ids']:
+            property = PropertyModel.find_by_id(id)
+            if(not property):
+                return{'message': 'Properties cannot be archived'}, 400
```

Comment:
```suggestion
            property = PropertyModel.find(id)
```
Using the find method will automatically return a 400 if the property cannot be found.